mimic-iv/concepts_postgres/medication/norepinephrine_equivalent_dose.sql

```diff
@@ -3,15 +3,27 @@ DROP TABLE IF EXISTS norepinephrine_equivalent_dose; CREATE TABLE norepinephrine
 -- This query calculates norepinephrine equivalent dose for vasopressors.
 -- Based on "Vasopressor dose equivalence: A scoping review and
 -- suggested formula" by Goradia et al. 2020.
+
+-- The relevant table makes the following equivalences:
+-- Norpeinephrine   - 1:1 - comparison dose of 0.1 ug/kg/min
+-- Epinephrine      - 1:1 [0.7, 1.4] - 0.1 ug/kg/min
+-- Dopamine         - 1:100 [75.2, 144.4] - 10 ug/kg/min
+-- Metaraminol      - 1:8 [8.3] - 0.8 ug/kg/min
+-- Phenylephrine    - 1:10 [1.1, 16.3] - 1 ug/kg/min
+-- Vasopressin      - 1:0.4 [0.3, 0.4] - 0.04 units/min
+-- Angiotensin II   - 1:0.1 [0.07, 0.13] - 0.01 ug/kg/min
+
 SELECT stay_id, starttime, endtime
     -- calculate the dose
+    -- all sources are in mcg/kg/min,
+    -- except vasopressin which is in units/hour
     , ROUND(CAST(
         COALESCE(norepinephrine, 0)
         + COALESCE(epinephrine, 0)
         + COALESCE(phenylephrine / 10, 0)
         + COALESCE(dopamine / 100, 0)
         -- + metaraminol/8 -- metaraminol not used in BIDMC
-        + COALESCE(vasopressin * 2.5, 0)
+        + COALESCE(vasopressin * 2.5 / 60, 0)
```

Comment:
Does this change need to be documented anywhere ( /60)?

Reply:
after rebase this should show correctly as a formatting change rather than a fix (the fix was in #1497)